src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    categoriesStatus: (state) => state.status,
```

Comment:
- [ ] @Bleu-Yves-Sopoude Great work on your categories reducer, however, the categoriesStatus action is supposed to check the `Under construction` status.
```suggestion
checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
```